.eslintrc.cjs

```diff
@@ -0,0 +1,97 @@
+module.exports = {
+	"root": true,
+	"parser": "@typescript-eslint/parser",
+	"parserOptions": {
+		"ecmaVersion": 6,
+		"sourceType": "module",
+		"ecmaFeatures": {
+			"impliedStrict": true,
+		},
+	},
+	"env": {
+		"browser": true,
+		"es6": true,
```

Comment:
Rather than specifying a specific version, is there a way to say "the newer version supported"?

Reply:
The purpose of this is to tell ESLint what predefined global variables are assumed to be available. My understanding is that Color.js supports down to Node v12 with legacy builds, and Node v16+ with the base build (and equivalent browsers). We could safely update this e.g. `es2020` based on that -- but it also doesn't hurt for this to be a lower version unless/until we start using cutting-edge ES functionality.

I'll update to `es2020` for now.